addons/web_editor/views/snippets.xml

```diff
@@ -464,74 +464,146 @@
                     using xpath to add non-animated shapes, also use xpath to
                     remove the t-if on the page. This is why individual options
                     also have that t-if.  -->
-                    <we-select-page class="o_we_basic_shapes" string="Basics" t-if="not no_animations">
-                        <we-button t-if="not no_animations" data-set-img-shape="web_editor/basic/bsc_square_1" data-select-label="Square 1" data-animated="true"/>
-                        <we-button t-if="not no_animations" data-set-img-shape="web_editor/basic/bsc_square_2" data-select-label="Square 2" data-animated="true"/>
-                        <we-button t-if="not no_animations" data-set-img-shape="web_editor/basic/bsc_square_3" data-select-label="Square 3" data-animated="true"/>
-                        <we-button t-if="not no_animations" data-set-img-shape="web_editor/basic/bsc_square_4" data-select-label="Square 4" data-animated="true"/>
-                        <we-button t-if="not no_animations" data-set-img-shape="web_editor/basic/bsc_square_5" data-select-label="Square 5" data-animated="true"/>
-                        <we-button t-if="not no_animations" data-set-img-shape="web_editor/basic/bsc_square_6" data-select-label="Square 6" data-animated="true"/>
-                        <we-button t-if="not no_animations" data-set-img-shape="web_editor/basic/bsc_square_rounded_1" data-select-label="Square Rounded 1" data-animated="true"/>
-                        <we-button t-if="not no_animations" data-set-img-shape="web_editor/basic/bsc_square_rounded_2" data-select-label="Square Rounded 2" data-animated="true"/>
-                        <we-button t-if="not no_animations" data-set-img-shape="web_editor/basic/bsc_organic_1" data-select-label="Organic Soft" data-animated="true"/>
-                        <we-button t-if="not no_animations" data-set-img-shape="web_editor/basic/bsc_organic_2" data-select-label="Organic Medium" data-animated="true"/>
-                        <we-button t-if="not no_animations" data-set-img-shape="web_editor/basic/bsc_organic_3" data-select-label="Organic Hard" data-animated="true"/>
+                    <we-select-page string="Geometrics">
+                        <we-button data-set-img-shape="web_editor/geometric/geo_square" data-select-label="Square" />
```

Comment:
> Also, I am wondering why are we are repeating the shape category in its name? Feels useless on top of being meaningless as it's mostly abbreviated: `cmp` for `composite`, `cmpn` for `composition`, `sld` for `solid`, `spl` for `special`.. It's a bit nitpicking but since we are refactoring everything, shouldn't we take the chance to clean that too? Unless there is a technical reason to do so but I don't think.

It seems that two svg having the same name, even in different folders can conflict with one another, having an abbreviation of their folder categoy makes it easier to not have duplicate svg names.